### PR TITLE
[XPU]Add conv1d fuse pass

### PR DESCRIPTION
**Pull request by wz1qqx**
### PR types

Performance optimization
### PR changes

OPs
### Description

- add conv1d_xpu_fuse_pass && conv1d_fusion kernel
- add phi elu xpu kernel
- fix link_xpu_op_max_pass

### CODE REVIEW

File: paddle/phi/backends/xpu/xpu2_op_list.cc [hunk 1]
```diff
@@ -164,6 +164,8 @@ XPUOpMap& get_kl2_ops() {
        XPUKernelSet({phi::DataType::FLOAT32, phi::DataType::FLOAT16})},
       {"conv2d",
        XPUKernelSet({phi::DataType::FLOAT32, phi::DataType::FLOAT16})},
+       {"conv1d_xpu",
+       XPUKernelSet({phi::DataType::FLOAT32, phi::DataType::FLOAT16})},
```
Comment (zhupengyang) [inline paddle/phi/backends/xpu/xpu2_op_list.cc]: - 格式需要调整，本地安装 pre-commit
- 按字母序排列

File: paddle/phi/api/yaml/fused_ops.yaml [hunk 2]
```diff
@@ -23,6 +23,16 @@
     func : add_layernorm_xpu
     data_type : x
 
+- op : conv1d_xpu
+  args : (Tensor x, Tensor x_max, Tensor filter, Tensor filter_max, Tensor bias, Tensor branch, Tensor branch_max, int64_t[] paddings, str padding_algorithm, int dilations, int strides, int groups, int act_type, float act_param)
```
Comment (zhupengyang) [inline paddle/phi/api/yaml/fused_ops.yaml]: paddings 类型定义为 int，和其它 conv 类算子保持一致

File: paddle/phi/backends/xpu/xpu2_op_list.cc (hunk 1 above)
Comment (wz1qqx) [reply paddle/phi/backends/xpu/xpu2_op_list.cc]: done

File: paddle/phi/api/yaml/fused_ops.yaml (hunk 2 above)
Comment (wz1qqx) [reply paddle/phi/api/yaml/fused_ops.yaml]: done

**Review by zyfncg — APPROVED** [review]

**Pull request merged**